src/main/resources/application-dev.yml

```diff
@@ -5,7 +5,8 @@ api:
   base:
     url:
       oauth: http://localhost:9090/auth
-      prisoner-search: http://localhost:8081
```

Comment:
do you need to change this constant to match?

https://github.com/ministryofjustice/hmpps-incident-reporting-api/blob/8cad7915df43e3b2b70312c57d117a15a73b8a6b/src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/integration/wiremock/PrisonerSearchMockServer.kt#L9

Reply:
they match already

Reply:
I've made everything match both docker, dev and wiremock